include/JetsonXavierGPIO/build.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
```

Comment:
I would of remove the example code maybe ? What do you think ? Do we really need to have the `build.sh` and `exampleGPIOApp.cpp ` files ?

Reply:
Seems logic, I'll remove them.